rcsb/workflow/wuw/PdbCsmImageWorkflow.py

```diff
@@ -0,0 +1,154 @@
+"""Images workflow functions that actually do the work."""
+
+import gzip
+import json
+import datetime
+import random
+from pathlib import Path
+import logging
+import subprocess
+
+logger = logging.getLogger(__name__)
+
+
+def get_pdb_list(args: dict) -> list:
```

Comment:
In all functions in this file: my style preference would be to use individual type parameters (e.g. `get_pdb_list(pdb_gz_path: str, update_all_images: bool)` instead of the `args` dictionary. Passing the args dictionary around requires to have the argument names hard-coded throughout the code, introducing a tight coupling between the CLI interface and the functions here.

Reply:
@josemduarte I originally passed them into variables but Douglas' ruff linter suggested I use something else. The reason being it only recommends passing <5 variables into a function for readability. I can change the linters settings of course but is there another potential solution you could recommend?

Reply:
Actually, I see you point is specifically about `get_pdb_list()` which does not have this problem since it doesn't require as many variables. What do you think of me changing `get_pdb_list()` and `get_csm_lsit()` to accepting variables and leaving `images_gen_lists()` and `images_gen_jpgs()` to accept the dict?

Reply:
As advice by @piehld let's follow the current convention of using `kwargs`

Reply:
I've changed each function to `**kwargs` now so I'll close this thread.